libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -165,9 +165,9 @@ class AP_BattMonitor
     /// consumed_wh - returns total energy drawn since start-up in watt.hours
     bool consumed_wh(float&wh, const uint8_t instance = AP_BATT_PRIMARY_INSTANCE) const WARN_IF_UNUSED;
 
-    /// capacity_remaining_pct - returns the % battery capacity remaining (0 ~ 100)
-    virtual uint8_t capacity_remaining_pct(uint8_t instance) const;
-    uint8_t capacity_remaining_pct() const { return capacity_remaining_pct(AP_BATT_PRIMARY_INSTANCE); }
+    /// capacity_remaining_pct - returns true if the percentage is valid and writes to percentage argument
+    virtual bool capacity_remaining_pct(int8_t &percentage, uint8_t instance) const WARN_IF_UNUSED;
```

Comment:
By convention instance goes first.

Reply:
In this file instance comes second for some reason, so I kept the signature consistent with the other ones (see definitions above this one).